RevenueCatUI/Templates/Template4View.swift

```diff
@@ -325,7 +325,7 @@ private struct PackageButton: View {
         // Placeholder to make sure consistent layout
         self.offerText(firstRow: "12",
                        secondRow: Localization.localized(packageType: .monthly,
-                                                         locale: self.locale))
+                                                         locale: self.locale) ?? "Monthly")
```

Comment:
Oh this should be `Localization.localizedDuration(for: .monthly, locale: self.locale)` instead

Reply:
Just pushed this 😇